pages/_app.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import { AppProps } from "next/app";
+import { Provider } from "react-redux";
+import { configureStore } from "../src/store";
+import Head from "next/head";
+
+const store = configureStore();
+
+export default function MyApp({ Component, pageProps }: AppProps) {
+  return (
+    <>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
why isnt the head portion in _document?

Reply:
The viewport and title tags cannot be placed in _document. See [here](https://stackoverflow.com/questions/65832820/next-js-viewport-meta-tags-should-not-be-used-in-document-jss-head)

Reply:
ah ok interesting, didnt remember this 👍